src/components/autocomplete/Autocomplete.vue

```diff
@@ -38,9 +38,16 @@
                     class="dropdown-content"
                     v-show="isActive"
                     :style="contentStyle">
-                    <div v-if="hasHeaderSlot" class="dropdown-item">
+                    <a
+                        v-if="hasHeaderSlot"
+                        class="dropdown-item"
+                        role="button"
+                        tabindex="0"
+                        :class="{ 'is-hovered': 'header' === hovered }"
+                        @click="setSelected('header', undefined, $event)"
```

Comment:
Why don't you emit 'select-header' instead of call `setSelected` ?

Reply:
Your question made me realize that setSelected was probably not the best function to be handling this event since it won't actually be setting a value. I went ahead and made a function for it instead that handles both the confirm keys and click event.